### PR TITLE
Improve: New PR Template

### PR DESCRIPTION
**Pull request by regalstreak**
## Proposed changes

Let's get the PR template a fresh look. We would need suggestions from maintainers on what more or what changes would be required in the PR template. The goal should be to get as much value out of the PR description as possible. (Same as on React.Native project)

## Closes issue(s)

#16967 

## How to test or reproduce

Open the document on a markdown editor.

### Old
image

### New
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by rodrigok** [comment]
Thanks @regalstreak 

Can you add the type of change `improvement`? Which improves an existent feature.

Would be nice to have a section to write a text to be included on changelog as you can see [here](https://github.com/RocketChat/Rocket.Chat/releases/tag/3.1.0-rc.0) on `Translation via MS translate` item.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
 <!-- [BREAK] For pull requests including breaking changes -->
 
-<!-- INSTRUCTION: Inform the issue number that this PR closes, or remove the line below -->
-Closes #ISSUE_NUMBER
+## Proposed changes
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue below. -->
 
-<!-- INSTRUCTION: Link to a https://github.com/RocketChat/docs PR with added/updated documentation or an update to the missing/outdated documentation list, see https://rocket.chat/docs/contributing/documentation/  -->
+## Closes issue(s)
+<!-- Link the issues being closed by this PR. For example, you can use #594 if this PR closes issue number 594 -->
```
Comment (rodrigok) [inline .github/PULL_REQUEST_TEMPLATE.md]: Will this close the issues without the `Closes` prefix?

Comment (regalstreak) [reply .github/PULL_REQUEST_TEMPLATE.md]: This is just meant to tag the issues that could be related to the PR. I will remove the `Closes` prefix

**Comment by regalstreak** [comment]
@rodrigok can you review again? Have added the changes

**Pull request merged**